packages/editor/src/components/post-template/panel.js

```diff
@@ -53,14 +53,17 @@ export default function PostTemplatePanel() {
 		return canCreateTemplates;
 	}, [] );
 
-	const canViewTemplates = useSelect( ( select ) => {
-		return (
-			select( coreStore ).canUser( 'read', {
-				kind: 'postType',
-				name: 'wp_template',
-			} ) ?? false
-		);
-	}, [] );
+	const canViewTemplates = useSelect(
+		( select ) => {
+			return isVisible
```

Comment:
Could you provide some more detail on why this works?

Reply:
The isVisible will be always false for Templates and Template parts, so we don’t need to run or use can view check.

Which fixes the mentioned issue with Duplicate action.

Those post types don’t have templates for understandable reasons anyway 😁

Reply:
The whole thing feels a bit convoluted, why do we have multiple store selectors instead of resolving something like `showClassicThemeControl` inside of a single one?